diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back-end is where things happens behind the scenes ... it is were the processing occurs ..
```

Comment:
Most importantly it is where our data gets stored, that ensures we are able to save data that will persist between sessions and different users.